doc/js/rose-doc.js

```diff
@@ -369,12 +369,15 @@ $(function () {
                 pathStr = $(this).attr("id");
                 var headings = $(this).find("h2, h3, h4, h5, h6");
                 var k;
+                var l;
```

Comment:
Can we have something else as index variable? `l` can easily be mistaken as `1`.